Funky-Chickens/workFORCEmanagement
# Edit employee progs

Changed:
Now the user can delete training programs FROM AN EMPLOYEE's profile - meaning they aren't attending that
(May need to write in a check to change/delete historical ... may not?)
We can also add a training to an employee from their detail view

Because:
This is required by ticket #4

To test:
Run npm start and navigate to http://localhost:4000/employees.
Choose an employee and try adding and deleting their training programs.
Check database for confirmation in psql.
